Code/GraphMol/SubstructLibrary/CMakeLists.txt

```diff
@@ -1,19 +1,32 @@
+
+
+
+
+if(RDK_USE_BOOST_SERIALIZATION AND Boost_SERIALIZATION_LIBRARY)
+    set(RDKit_SERIALIZATION_LIBS ${Boost_SERIALIZATION_LIBRARY})
```

Comment:
To be consistent with the rest of the code, please add `    ADD_DEFINITIONS("-DRDK_USE_BOOST_SERIALIZATION")` here instead of using `RDConfig.h` below.

Reply:
This kind of defeats the purpose doesn't it?  I.e. if anyone else imports this code in C++, this definition won't be configured properly and might cause linker issues.  It *might* be okay now that I have non-intrusive serialization, but in general any c++ dev should not have to worry about how cmake was configured.

We can fix the warnings in RDConfig.h by the way.

Reply:
I don't think it defeats the purpose. If I'm writing code that uses an installed copy of the RDKit, I can `#include <RDConfig.h>` before I include any other RDKit headers and everything should be fine.
Am I missing something?